### DISCUSSION TITLE
Plot without NA

### DISCUSSION THREAD

**Discussion by nrblsffspc** [jazznbass/scan#76] [Q&A] [answered]
Hi dear team,

I have a probably easy question about plots without NA's: for a presentation of the data, I would like to show **one plot with the missings** and **one without**, for simplicity. I wonder if I have overlooked a simple function regarding this so far, or if I need to solve this via a subset? 

Lots of greetings

**Comment by jazznbass** [comment, marked as answer]
I am not sure what you mean by showing the plot without missings.

Basically, you can drop NAs or you can interpolate NAs.

(Did you read the chapter on missing values?
https://jazznbass.github.io/scan-Book/describe-and-manipulate-single-case-data-frames.html?q=missing#missing-values


Here is an example applying both strategies:

```r
case2 <- scdf(c(3,6,2,4,3,5,2,NA,3,2, 6,7,5,8,6,NA,4,8,5,6), 
              phase_design = c(A = 10, B = 10), name = "NAs")
case3 <-  subset(case2, !is.na(values))
case4 <- fill_missing(case2)
names(case3) <- "Dropped NAs"
names(case4) <- "Interpolated NAs"
ex <- c(case2, case3, case4)
```

![image](https://user-images.githubusercontent.com/50012671/191549133-849468e4-3ea9-43e2-a1d1-77e49b5ffd0a.png)

Note: The above plot has been created with the new scplot package (but the standard plot still works). You could try out the new scplot package (still in development):

https://jazznbass.github.io/scan-Book/scplot---adavanced-plotting-functions-for-single-case-data.html#scplot---adavanced-plotting-functions-for-single-case-data

Best,
Jürgen
